src/window.js

```diff
@@ -78,6 +79,8 @@ export default function Window({ application }) {
 
   const placeholders = readDemo("Welcome");
 
+  const documentation_viewer = DocumentationViewer({window,application})
```

Comment:
```suggestion
  DocumentationViewer({window, application})
```
`documentation_viewer`  isn't used

consider moving this line next to `Libray()`